### DISCUSSION TITLE
Upgrading Apache Airflow from version 2.1.4 to 2.2.1

### DISCUSSION THREAD

**Discussion by Elebath** [apache/airflow#19444] [Q&A]
### Apache Airflow version

2.1.4

### Operating System

Windows

### Deployment

Docker-Compose

### What happened

Following the tutorial on how to upgrade Airflow versions (https://airflow.apache.org/docs/apache-airflow/stable/upgrading-from-1-10/index.html) gave the following message:

Airflow found incompatible data in the task_instance table in the metadatabase, and has moved them to _airflow_moved__2_2__task_instance during the database migration to upgrade. Please inspect the moved data to decide whether you need to keep them, and manually drop the _airflow_moved__2_2__task_instance table to dismiss this warning.

### What you expected to happen

I could not find a way to get past this. No further way to move pass that was shown. Maybe this is normal.

### How to reproduce

I am a beginner and I am not so sure. 

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
@Elebath  - please take a look at #19453 to see if the additional docs being added there would answer your question (maybe you can propose some improvements to the message as well)

**Comment by adaezebestow** [comment]
How can I manually drop the _airflow_moved__2_2__task_instance table to dismiss this warning?

**Reply by potiuk** [reply]
https://airflow.apache.org/docs/apache-airflow/stable/installation/upgrading.html#post-upgrade-warnings

**Reply by potiuk** [reply]
BTW. Did you have a chance to read the #19453 which was adding this @adaezebestow ? Or what is the reason you have not seen it before? Just wonder if this was unclear, or difficult to follow ?

**Reply by adaezebestow** [reply]
@potiuk I didn't see the link you sent me referenced earlier in this thread. My understanding is the discussion was focused on making the error message clearer. I was able to resolve the issue by following the steps below:

1. Exec into any of the Airflow pods - Webserver or scheduler is fine, for example:
`kubectl exec -it <your-webserver-pod> python`

2. Run the following commands in a python shell:
```
from airflow.settings import Session
session = Session()
session.execute("DROP TABLE _airflow_moved__2_2__task_instance")
session.commit()
```

**Reply by potiuk** [reply]
Cool

**Reply by davidavdav** [reply]
Yay! This worked, versus not worked: `airflow db shell`, `drop table _airflow_moved__2_2__task_instance` because I guess psql has a different syntax than mysql.

**Reply by potiuk** [reply]
Cool. PR with improving the page woudl be nice! It's as easy as clicking "Suggest a change on this page" and submitting a PR.

**Reply by adaezebestow** [reply]
@potiuk I can't quite figure out how to suggest a change on the post-upgrade warnings page: https://github.com/apache/airflow/blob/main/docs/apache-airflow/installation/upgrading.rst

**Reply by potiuk** [reply]
It's from the documetation page (look at the bottom-right button) https://airflow.apache.org/docs/apache-airflow/stable/installation/upgrading.html
